packages/guardian-api/src/resources/users.ts

```diff
@@ -1,30 +1,5 @@
-import { Contracts } from "@arkecosystem/core-api";
-import { Container } from "@arkecosystem/core-kernel";
+import { Interfaces } from "@protokol/guardian-transactions";
 
-@Container.injectable()
-export class UserResource implements Contracts.Resource {
```

Comment:
Why did you remove Contracts.Resource class, each response should have one, so we follow the same convention as in nfts and in core-api

Reply:
Based on this PR: https://github.com/ArkEcosystem/core/pull/3931 resources aren't transformer classes anymore.

Reply:
ok thanks for letting me know